src/libs/processReportIDDeeplink/index.ts

```diff
@@ -0,0 +1,5 @@
+import getReportIDfromUrl from './getReportIDfromUrl';
+
+export default function processReportIDDeeplink(url: string) {
+    return getReportIDfromUrl(url);
```

Comment:
why not placing the code of `getReportIDfromUrl` here?

in case of keeping it, shouldn't be `getReportIDFromUrl`?

Reply:
Done - 1310235.